scripts/zones/Quicksand_Caves/mobs/Centurio_X-I.lua

```diff
@@ -11,7 +11,7 @@ function onMobInitialize(mob)
 end;
 
 function onMobSpawn(mob)
-    mob:addMod(MOD_SILENTRES, 35);
+    mob:addMod(MOD_SILENCERES, 35);
```

Comment:
good catch, I wonder how that didn't get reported as a crash before..

Also occurs to me we have these mods to make it resist sleeps when its race/job should already have given is sleep and dark resistances to handle that. Someone will have to look into that someday /shrug (not anything to do with your pr, I just noticed it coz we're here, you can ignore it)